lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -222,7 +229,7 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
         );
     }
 
-    onAttachFileClicked(nodeSelector: any) {
+    onShowFile(nodeSelector: any) {
```

Comment:
can you provide typing here instead of "any" please

Reply:
renamed to `onViewNode(node: Node)`.